src/styles/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 20px 10px;
+  max-width: $maxWigthBox;
+  line-height: $square;
+  text-align: center;
+
+  &__day {
+    background-color: #eee;
+    height: $square;
+    width: $square;
+    border: 1px solid black;
+    box-sizing: border-box;
+    transition: 500ms;
```

Comment:
do not add `transition` for `all` properties, it is bad performance-wise. Specify only the properties you need.
check out point 9 with examples in checklist